app/src/main/java/org/mifos/mobilebanking/utils/MFDatePicker.java

```diff
@@ -78,12 +81,17 @@ public static String getDatePickedAsString() {
     @Override
     public Dialog onCreateDialog(Bundle savedInstanceState) {
 
+        int year = selectedDate.get(Calendar.YEAR);
+        int month = selectedDate.get(Calendar.MONTH);
+        int day = selectedDate.get(Calendar.DAY_OF_MONTH);
+
         DatePickerDialog dialog = new DatePickerDialog(getActivity(),
                 R.style.MaterialDatePickerTheme,
                 this,
-                calendar.get(Calendar.YEAR),
-                calendar.get(Calendar.MONTH),
-                calendar.get(Calendar.DAY_OF_MONTH));
+                year,
+                month,
+                day);
+
```

Comment:
Here too.

Reply:
There is a way to do in the same pull request.
After removing the extra lines (which I commented ) while committing just add `amend` method like this `git commit --amend` 
You can refer [this](https://es.atlassian.com/git/tutorials/rewriting-history#git-commit--amend) for more information

Reply:
okay thanks

Reply:
is the latest build fine?

Reply:
Fine